databases/week2/working-with-task.sql

```diff
@@ -0,0 +1,46 @@
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+  task(title, description, created, updated, due_date, status_id, user_id)
+VALUES
+  ("Do homework", "must be done before friday night", "2022-03-15 10:01:16", "2022-03-15 10:01:16", "2022-03-18 10:01:16", 1, 6);
```

Comment:
@Elia-Git  Hi, thanks for your review. I got why my queries not worked like it have to regarding to task. This because of I had db with the same data and almost the same scheme like in the task. We created it on first lesson. But it has not table `user_task`. That's why my queries worked for me when I commitid homework. I changed queries regarding to db schema that is given in homework task description. Sorry for this.